lib/inventory/controller.go

```diff
@@ -335,12 +323,10 @@ func (c *Controller) handleControlStream(handle *upstreamHandle) {
 	}
 }
 
-// instanceHeartbeatsEnabledEnv checks if instance heartbeats have been enabled via unstable
-// env var. This is a placeholder until we've sufficiently improved instance heartbeat
-// backend performance to reenable them by default.
-// TODO(tross,fspmarshal): remove this once issues with etcd stability are resolved.
-func instanceHeartbeatsEnabledEnv() bool {
-	return os.Getenv("TELEPORT_UNSTABLE_ENABLE_INSTANCE_HB") == "yes"
+// instanceHeartbeatsDisabledEnv checks of instance heartbeats have been explicitly disabled
+// via environment variable.
+func instanceHeartbeatsDisabledEnv() bool {
```

Comment:
Should we keep a TODO here to eventually remove this?

Or maybe rename the env var? If we are considering instance heartbeats stable enough to enable by default, it might be confusing for Teleport to change behavior based on a variable named `UNSTABLE_ENABLE_INSTANCE_HB`.